dts/bindings/usb/usb-ep.yaml

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2018, I-SENSE group of ICCS
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: USB Endpoints' properties
+id: USB-EP
+version: 0.1
+
+description: >
+    This binding gives number of endpoints that the USB hardware supports
+
+properties:
+    num-bidir-endpoints:
```

Comment:
How will this configuration be used?

Reply:
It is the number of bi-directional endpoints. You could check how it is used in this PR (stm32f4.dtsi file) and PR #5163.